### DISCUSSION TITLE
Request: Support for the NIF format of "Star Trek: Bridge Commander"

### DISCUSSION THREAD

**Discussion by FekLeyrTarg** [BadDogSkyrim/PyNifly#154] [Ideas]
Hello people,

an acquaintance of mine has pointed me to this project.
I find it awesome that people create new I/O plugins for Blender so more people can create new stuff for their favorite game.
So I appreciate your efforts very much.

I'm a huge fan of "Star Trek: Bridge Commander", which was released in 2002 two months before "Morrowind" and uses an early version of the Gamebryo Engine.
Starships, stations, bridge sets and characters all use a variant of the NIF format.

Unfortunately, no current NIF plugin for Blender supports STBC.
There's a plugin for Milkshape but it only supports ships and stations and it does break normal smoothness.
Another plugin was released for 3D Studio Max 3.1 as part of the BC SDK by the original developers.
These days however it is difficult to find a copy of that version on CD-ROM and it is questionable whether or not it would run on Windows 10.
Also NifSkope discontinued supporting STBC after version 1.6 (ish)

So my big wish for PyNifly is support for STBC's NIF format for ships, stations, bridges and characters.
That way more people who are interested in creating content for that game will be able to do so, potentially revitalizing interest in BC modding.

I and many others in the BC modding community would be very thankful for that. :-)

**Comment by BadDogSkyrim** [comment]
Yeah,  not likely I can do this. I'm working to get animations done for the current games before Starfield arrives and sucks all the oxygen out of modding for other games and once it does, I'll be modding for Starfield too. I'm not likely to be able to prioritize this.
